source/docs/articles/local/cli.md

```diff
@@ -11,7 +11,7 @@ The Terminus project's second version is developed and supported at [https://git
 Terminus is in a pre-release state with constant development. The commands in the repo may change and not all functionality will work for all use cases.
 
 ## Installation
-Installation instructions are maintained at the [project wiki](https://github.com/pantheon-systems/cli/wiki/Installation "Pantheon CLI wiki, installation page on Github").
+Installation instructions are maintained at the [project wiki](https://github.com/pantheon-systems/cli/blob/master/README.md#installation "Pantheon CLI wiki, installation page on Github").
 ## Use
```

Comment:
Suggest linking to https://github.com/pantheon-systems/cli#installation instead but I don't have a huge preference, defer to @nataliejeremy